src/v2/modules/invoice-accounts/controller.js

```diff
@@ -16,5 +16,13 @@ const getInvoiceAccounts = async request => {
   return results.map(mappers.invoiceAccount.currentAddressOnly);
 };
 
+const postInvoiceAccount = async (request, h) => {
```

Comment:
By using the entityHandlers.createEntity function in the route, this controller handler won't be required.

You will have to write and invoiceAccountService.createInvoiceAccount function though, similar to https://github.com/DEFRA/water-abstraction-tactical-crm/blob/develop/src/v2/services/contacts.js#L7. 

At this point business rule validation of the payload is done, leaving the route validation just determine if the request payload is suitable to continue with.